app/models/linkage_group.rb

```diff
@@ -0,0 +1,14 @@
+class LinkageGroup < ActiveRecord::Base
+
+  has_and_belongs_to_many :linkage_maps,
+                          join_table: 'map_linkage_group_lists',
+                          foreign_key: 'linkage_group_id',
+                          association_foreign_key: 'linkage_map_id'
```

Comment:
I assume you do not want to rename these keys to traditional SQL values? For me it's ok since I hope to work with relations only, but could be difficult in the future when debugging problems directly on the DB...